core/src/main/java/org/apache/calcite/sql/dialect/MssqlSqlDialect.java

```diff
@@ -132,6 +132,17 @@ public void unparseOtherFunction(SqlWriter writer, SqlCall call, int leftPrec, i
       call.operand(0).unparse(writer, leftPrec, rightPrec);
       writer.endFunCall(logFrame);
       break;
+    case "ROUND":
+      if (call.getOperandList().size() < 2) {
+        final SqlWriter.Frame roundFrame = writer.startFunCall("ROUND");
+        call.operand(0).unparse(writer, leftPrec, rightPrec);
+        writer.print(",");
+        writer.sep("0");
```

Comment:
Use SqlWriter.sep() for ',' as it is a separator
and use SqlWriter.print() for '0' as it is a literal

Reply:
Done